tests/test_graphical_units/test_axes.py

```diff
@@ -171,14 +171,28 @@ def test_t_label(scene):
     scene.add(axes, func, t_label)
 
 
+@frames_comparison
+def test_get_area_with_riemann_rectangles(scene):
```

Comment:
Could you add a test for the new version too?

Reply:
It should be just below that one